src/main/java/io/cdap/e2e/pages/actions/CdfStudioActions.java

```diff
@@ -355,6 +355,21 @@ public static void connectSourceAndSink(String source, String sourceTitle, int i
     connectSourceAndSink(source, sourceTitle, sink, sinkTitle);
   }
 
+  /**
+   * Connect two plugin nodes
+   *
+   * @param source     Source Plugin name
+   * @param target     Target Plugin name
+   * @param targetType Target Plugin name
+   */
+  public static void connectTwoPluginNodes(String source, String target, String targetType) {
```

Comment:
param names source and target tend to confuse with source and sink plugins, can we rename them to fromPlugin and toPlugin?

Reply:
Addressed. Does the `Build e2e tests` workflow have to pass before merging? Seems like it's been consistently failing lately.

Reply:
yeah, ideally it should pass before merge. Currently it is failing due to some issue in running preview in sandbox.